lib/git_reflow/commands/review.rb

```diff
@@ -6,15 +6,38 @@
   c.flag [:t, :title], default_value: 'last commit message'
   c.flag [:m, :message], default_value: 'title'
   c.action do |global_options,options,args|
-    review_options = {
-      'base' => args[0],
-      'title' => global_options[:title],
-      'body' => global_options[:message]
-    }
 
-    review_options['title'] ||=  GitReflow.get_first_commit_message
-    review_options['body']  ||=  review_options['title']
+    pull_request_msg_file = '.git/GIT_REFLOW_PR_MSG'
```

Comment:
just remembered there was an PR open with an issue of relative paths... use `git_root_dir = run('git rev-parse --show-toplevel').strip` and prepend it to your path here